<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.cache.interceptor.CacheSyncFailureTests" tests="5" skipped="0" failures="0" errors="0" timestamp="2019-12-09T02:20:30" hostname="DESKTOP-246NTI5" time="0.061">
  <properties/>
  <testcase name="syncWithTwoGetOperations()" classname="org.springframework.cache.interceptor.CacheSyncFailureTests" time="0.018"/>
  <testcase name="unlessSync()" classname="org.springframework.cache.interceptor.CacheSyncFailureTests" time="0.01"/>
  <testcase name="severalCachesWithResolvedSync()" classname="org.springframework.cache.interceptor.CacheSyncFailureTests" time="0.011"/>
  <testcase name="severalCachesSync()" classname="org.springframework.cache.interceptor.CacheSyncFailureTests" time="0.01"/>
  <testcase name="syncWithAnotherOperation()" classname="org.springframework.cache.interceptor.CacheSyncFailureTests" time="0.01"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
